wgpu-hal/src/metal/mod.rs

```diff
@@ -25,12 +25,14 @@ mod device;
 mod surface;
 mod time;
 
-use std::{
-    fmt, iter, ops,
-    ptr::NonNull,
-    sync::{atomic, Arc},
-    thread,
-};
+use std::{fmt, iter, ops, ptr::NonNull, sync::Arc, thread};
+mod atomic {
+    #[cfg(feature = "portable-atomic")]
+    pub use portable_atomic::AtomicU64;
+    #[cfg(not(feature = "portable-atomic"))]
+    pub use std::sync::atomic::AtomicU64;
+    pub use std::sync::atomic::Ordering;
+}
```

Comment:
I think it's fine to change these `use`s to be just on the types themselves, so we don't need to do this module trick.

Reply:
done